funnel/views/account.py

```diff
@@ -50,6 +52,20 @@
 from .sms import send_phone_verify_code
 
 
+def md5sum_or_blake2b_b58(text):
```

Comment:
This function is temporary until MD5 links are obsolete in a week or so. It can be removed then. We will need a more robust mechanism for casting from binary to base58 bidirectionally however, perhaps an equivalent of `uuid_b58`'s comparator in Coaster.